### PR TITLE
[SecurityBundle] don't validate IP addresses from env var placeholders

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 4.3
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #32108
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
I'm not an expert in this ... but changes look OK to me. Thanks Christian.

File: src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php [hunk 1]
```diff
@@ -731,20 +731,30 @@ private function createExpression($container, $expression)
...
+    private function createRequestMatcher(ContainerBuilder $container, $path = null, $host = null, int $port = null, $methods = [], array $ips = null, array $attributes = [])
     {
         if ($methods) {
             $methods = array_map('strtoupper', (array) $methods);
         }
 
-        $id = '.security.request_matcher.'.ContainerBuilder::hash([$path, $host, $port, $methods, $ip, $attributes]);
+        if (null !== $ips) {
+            foreach ($ips as $ip) {
+                $container->resolveEnvPlaceholders($ip, null, $usedEnvs);
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php]: missing reset before the call: `$usedEnvs = [];`

Comment (xabbuh) [reply src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php]: fixed

**Review by nicolas-grekas — APPROVED** [review]

**Review by Simperfit — APPROVED** [review]

**Comment by Simperfit** [comment]
I've tested it and it works.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Comment by Wirone** [comment]
When it will be available in stable release? We need this fix to upgrade our dependencies, but I prefer not to use `"symfony/security-bundle": "4.3.*@dev",` in our `composer.json`.

**Comment by xabbuh** [comment]
@Wirone Patch releases are published every four weeks regularly.

**Pull request merged**